code/modules/mob/mob_helpers.dm

```diff
@@ -14,6 +14,9 @@
 	if(isStunned() || restrained())
 		return 1
 
+/mob/proc/isHandcuffed() // Simple mobs don't have hands ergo aren't cuffed
```

Comment:
Sorry if this is out of scope. It was the best way I found to do the checks.

Reply:
You probably want to use `restrained` instead of defining a new proc.